src/Agenda.js

```diff
@@ -56,6 +56,34 @@ class Agenda extends React.Component {
     events = events.filter(event =>
       inRange(event, date, end, this.props)
     );
+
+    if (events.length === 0) {
+      return (
+        <div className='rbc-agenda-view'>
+          <table ref='header'>
+            <thead>
+              <tr>
+                {agendaLength > 0 && <th className='rbc-header' ref='dateCol'>
+                  {messages.date}
+                </th>}
+                <th className='rbc-header' ref='timeCol'>
+                  {messages.time}
+                </th>
+                <th className='rbc-header'>
+                  {messages.event}
+                </th>
+              </tr>
+            </thead>
+          </table>
+          <div className='rbc-empty-agenda-content' ref='content' style={{height: '100vh', display: 'flex', flexDirection: 'column', justifyContent: 'center'}}>
```

Comment:
Why not put these styles into a stylesheet with a class? This will make it hard for other users to override the style.